dts/arm/nordic/nrf52805.dtsi

```diff
@@ -217,13 +217,55 @@
 			status = "okay";
 		};
 
+		swi0: swi@40014000 {
+			compatible = "nordic,nrf-swi";
```

Comment:
These nodes will also be added in #44276, and it seems that you don't need to have them in this PR, so please drop this change, too.